packages/ui-signer/src/Modal.tsx

```diff
@@ -189,6 +194,31 @@ class Signer extends React.PureComponent<Props, State> {
     });
   }
 
+  onKeyDown = (event: any): void => {
+    let buttonEls: any = [];
+    let inputList: any = [];
+    let buttonFilteredEls: any = [];
+
+    if (event.keyCode === 13) {
+      buttonEls = document.getElementsByTagName('button');
```

Comment:
We really, really should not be mixing DOM access with React. The React-way is to use references.